src/platform/selfsign_openssl.c

```diff
@@ -216,7 +216,7 @@ QuicTlsGenerateSelfSignedCert(
     fclose(Fd);
     Fd = NULL;
 
-    Fd = fopen(CertFileName, "wb");
+    Fd = fopen(CertFileName, "wbe");
```

Comment:
This is also called from windows, which doesn't accept the e flag. Since the e flag is specific only to gclibc, I would very much prefer this not be in here at all, because otherwise mac will break when we get to it too.

Reply:
Darwin has support for `e`; but I have removed it anyway in the latest push.